src/Microsoft.AspNetCore.Sockets/WebSockets.cs

```diff
@@ -22,6 +23,7 @@ public class WebSockets : IHttpTransport
         private readonly WebSocketOpcode _opcode;
         private readonly ILogger _logger;
 
+        public WebSockets(Connection connection, Format format) : this(connection, format, loggerFactory: null) { }
```

Comment:
Ok, wasn't sure what the best practice here was since I saw `NullLogger`. Will update.